mmdet/datasets/transforms/geometric.py

```diff
@@ -225,10 +225,10 @@ class ShearX(GeomTransform):
         level (int, optional): The level should be in range [0, _MAX_LEVEL].
             If level is None, it will generate from [0, _MAX_LEVEL] randomly.
             Defaults to None.
-        min_mag (float): The minimum magnitude for the horizontal shear.
+        min_mag (float): The minimum angle for the horizontal shear.
```

Comment:
```suggestion
        min_angle (float): The minimum angle for the horizontal shear.
```

Reply:
All geometric transformations need to maintain the same parameter names `min_mag` and `max_mag`， because they inherit the same base class

Reply:
okay